src/pages/ProfileForm/index.tsx

```diff
@@ -89,15 +90,11 @@ function CompleteUserProfileForm(): JSX.Element | null {
     !isUserEmailDomainAssociatedWithAnyInstitution;
 
   /* If there is no financial institutions associated with user's email domain, use the 'Add Financial Institution' form instead */
-  const UserProfileForm = isNonAssociatedEmailDomain
-    ? CreateProfileForm
-    : Step1Form;
+  if (isNonAssociatedEmailDomain) {
+    return <Navigate replace to='/profile/complete/no-assocations' />;
```

Comment:
I think you also forgot this one...

```suggestion
    return <Navigate replace to='/profile/complete/no-associations' />;
```